internal/cmd/gocdk/internal/static/_assets/init/go.mod

```diff
@@ -2,4 +2,4 @@ module {{.ModulePath}}
 
 go 1.12
 
-require gocloud.dev v0.15.0
+require gocloud.dev v0.15.1-0.20190626205612-1012b9d3127e
```

Comment:
Note: I had to set this to a pseudo-version in order to load docstore. As soon as we have a real release with docstore we should update this.